docs/reading/good_enough_practices_in_scientific_computing_wilson_et_al.yml

```
@@ -0,0 +1,14 @@
+title: "Good Enough Practices in Scientific Computing"
+author: "Greg Wilson et al."
+availability:
+  - "`Free pdf <https://arxiv.org/pdf/1609.00037v2.pdf>`_"
```

https://doi.org/10.1371/journal.pcbi.1005510
```suggestion
  - "`https://doi.org/10.1371/journal.pcbi.1005510`_"
```

---

Link to final paper?